s3_upload/utils/upload.py

```
@@ -232,7 +232,7 @@
     return local_file, remote_object.get("ETag", "").strip('"')
 
 
-def _submit_to_pool(pool, func, item_input, items, **kwargs):
+def _submit_to_pool(pool, func, item_input, items, **kwargs) -> dict:
```

Ruff (0.7.0)

Missing type annotation for function argument `pool`

(ANN001)